lib/components/primitive-components/Trace/create-downward-net-label-ground-symbol.ts

```diff
@@ -48,52 +48,57 @@ export const createDownwardNetLabelGroundSymbol = (
   { db }: { db: CircuitJsonUtilObjects },
 ): void => {
   const offsets = calculateOffsets(port)
+
+  const isLeftOrRight =
+    port.facingDirection === "left" || port.facingDirection === "right"
+
+  const horizontalEndPos = {
+    x: anchorPos.x + offsets.horzPortDirection * offsets.schBoxHorzOffset,
+    y: anchorPos.y + offsets.vertPortDirectionOffset * offsets.schBoxVertOffset,
+  }
+
+  const groundSymbolPos = isLeftOrRight
+    ? {
+        x: horizontalEndPos.x,
+        y: horizontalEndPos.y - 0.5,
+      }
+    : horizontalEndPos
+
   db.schematic_net_label.insert({
     anchor_side: "top",
-    center: {
-      x: anchorPos.x + offsets.horzPortDirection * offsets.schBoxHorzOffset,
-      y:
-        anchorPos.y +
-        offsets.vertPortDirectionOffset * offsets.schBoxVertOffset,
-    },
+    center: groundSymbolPos,
     source_net_id: port.source_port_id!,
     text: schDisplayLabel!,
-    anchor_position: {
-      x: anchorPos.x + offsets.horzPortDirection * offsets.schBoxHorzOffset,
-      y:
-        anchorPos.y +
-        offsets.vertPortDirectionOffset * offsets.schBoxVertOffset,
-    },
+    anchor_position: groundSymbolPos,
     symbol_name: "ground_horz",
   })
 
-  db.schematic_trace.insert({
-    edges: [
-      {
-        from: { x: anchorPos.x, y: anchorPos.y },
-        to: {
-          x: anchorPos.x,
-          y: anchorPos.y + offsets.handleUp * offsets.schBoxVertOffset,
-        },
-      },
-      {
-        from: { x: anchorPos.x, y: anchorPos.y },
-        to: {
-          x: anchorPos.x + offsets.horzPortDirection * offsets.schBoxHorzOffset,
-          y:
-            anchorPos.y +
-            offsets.vertPortDirectionOffset * offsets.schBoxVertOffset,
-        },
+  const edges = [
+    {
+      from: { x: anchorPos.x, y: anchorPos.y },
+      to: {
+        x: anchorPos.x,
+        y: anchorPos.y + offsets.handleUp * offsets.schBoxVertOffset,
       },
-    ],
-    junctions: [
-      {
-        x: anchorPos.x + offsets.horzPortDirection * offsets.schBoxHorzOffset,
-        y:
-          anchorPos.y +
-          offsets.vertPortDirectionOffset * offsets.schBoxVertOffset,
-      },
-    ],
+    },
+    {
+      from: { x: anchorPos.x, y: anchorPos.y },
+      to: horizontalEndPos,
+    },
+  ]
+
+  if (isLeftOrRight) {
+    edges.push({
+      from: horizontalEndPos,
+      to: groundSymbolPos,
+    })
+  }
+
+  const junctions = [groundSymbolPos]
```

Comment:
sorry, I'm looking at it and I don't think you're supposed to have a junction here either. A junction is where two traces cross and connect, but in this case there is no other trace

```suggestion
  const junctions = []
```